src/livestreamer/plugins/itvplayer.py

```diff
@@ -0,0 +1,183 @@
+import re
+from urlparse import urlparse, urlunparse, urlsplit
```

Comment:
Sure, too used to python 2... Do you want a separate pull for the compat change?